ui/src/environments/environment-type.ts

```diff
@@ -15,6 +15,9 @@ export interface Environment {
   // Example value: 'Test' would cause the following full title:
   // "Homepage | [Test] All of Us Research Workbench"
   displayTag: string;
+  // Indicates if the displayTag should be shown in the web app. If it is true,
+  // a small label will be added under the "All of Us" logo in the header.
+  shouldShowDisplayTag: boolean;
```

Comment:
I'm not sure this extra flag is necessary.

We already have a convention where in production the displayTag is an empty string, and app/component.ts (L89) then checks for a non-empty value in order to decide whether to add a prefix to the tab title.

It might be more consistent to just use the same convention for the in-app display tag as well.

Counterpoint: I can see an argument that having explicit "shouldShowDisplayTag" is clearer to understand. If we believe that's the case, then we should be consistent in how we're applying this logic, and use shouldShowDisplayTag in L89 of app/component.ts to control whether the prefix is included in the tab title.

Your call – after reading this through, my slight preference would be for adding explicit control to app/component.ts, but I can see it either way.

Reply:
I don't have a strong preference either but I think I prefer to keep it separate since I almost view the `displayTag` as an environment name. I think this opens us up to using it as such since its no longer tied to being an empty string.